docs/usage.rst

```diff
@@ -179,6 +179,12 @@ For more information about that, see: https://xkcd.com/1179/
 Unless otherwise noted, we display local date and time.
 Internally, we store and process date and time as UTC.
 
+Common options
+~~~~~~~~~~~~~~
+
+All Borg commands share these options:
+
+.. include:: usage/--common.rst.inc
```

Comment:
Ah yes, the idea was to have something that can't ever clash with a command name, but `--something` files are also notoriously annoying to handle on the command line. Changed it ;)